DDDetectors/compact/SiD.xml

```diff
@@ -44,8 +44,8 @@
     </limitset>
     <limitset name="SiTrackerBarrelRegionLimitSet">
       <limit name="step_length_max" particles="*" value="5.0" unit="mm" />
-      <limit name="track_length_max" particles="*" value="5.0" unit="mm" />
-      <limit name="time_max" particles="*" value="5.0" unit="ns" />
+      <!-- <limit name="track_length_max" particles="*" value="5.0" unit="mm" /> -->
+      <!-- <limit name="time_max" particles="*" value="5.0" unit="ns" /> -->
```

Comment:
Why leave this commented? Then add some comment on what happens if you enable.

Reply:
I agree with Marko. Either a comment should be added or the Geant4 default value which is presumably big should be put in.

Reply:
I will remove these lines.